utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -230,11 +230,8 @@ object SwaggerObject {
 
     val additionalProperties = schema.getAdditionalProperties match {
       case null => AdditionalPropertiesEnabled(SwaggerAny)
-      case additionalPropertyEnabled if additionalPropertyEnabled == true => AdditionalPropertiesEnabled(SwaggerAny)
-      case additionalPropertyEnabled if additionalPropertyEnabled == false => AdditionalPropertiesDisabled
-      case additionalPropertySchema: Schema[_] =>
-        val typed = AdditionalPropertiesEnabled(SwaggerTyped(additionalPropertySchema, swaggerRefSchemas, usedRefs))
-        typed
+      case schema: Schema[_] if schema.getBooleanSchemaValue == false => AdditionalPropertiesDisabled
+      case schema: Schema[_] => AdditionalPropertiesEnabled(SwaggerTyped(schema, swaggerRefSchemas, usedRefs))
```

Comment:
do we change behaviour here?

Reply:
Tests:
https://github.com/TouK/nussknacker/blob/73400320cd4d7dbc89110ef2a0f0d7ca0fe619c6/utils/json-utils/src/test/scala/pl/touk/nussknacker/engine/json/SwaggerBasedJsonSchemaTypeDefinitionExtractorTest.scala#L15:L35